src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -226,6 +222,13 @@ export default function RequestedAppointmentDetailsPage() {
           )}
           {!provider && <span>No provider selected</span>}
         </>
+      ) : (
+        <>
+          <h2 className="vaos-appts__block-label vads-u-margin-bottom--0 vads-u-margin-top--2">
+            Preferred type of appointment
+          </h2>
+          {appointment.vaos.requestVisitType}
```

Comment:
In this component we set a lot of these `vaos` object properties to variables. They're above the this final return statement. We could do that for this property as well.

Reply:
Good call.